frontend/src/components/TicketDetails.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { StateContext } from '../State';
+
+const TicketDetails = () => {
+    const context = React.useContext(StateContext);
+
+    let specificTicket = context.tickets.queue.filter(i => i.id == context.tickets.selected)[0]
```

Comment:
Consider using `===` instead of `==` here.  Deep and shallow equality differ slightly in how they compare two values, and it's generally best practice to use `===` wherever possible.